src/components/Scaler/index.js

```diff
@@ -59,6 +59,39 @@ class Scaler extends Component {
   }
 
   componentDidUpdate(prevProps) {
+    if (!isEqual(prevProps.dataContext.series, this.props.dataContext.series)) {
+      const domainsByItemId = {};
+      const subDomainsByItemId = {};
+
+      const updateDomainsForItem = item => {
+        domainsByItemId[item.id] = {
+          [Axes.time]: this.props.dataContext.timeDomain,
```

Comment:
seems like `DataProvider` `timeDomain` prop is not Required so it could be not set, shouldn't we set some default in that case?

Reply:
Done. I also flipped the order of the other domains (and subdomains).